### DISCUSSION TITLE
"Something went wrong" when creating a ticket

### DISCUSSION THREAD

**Discussion by nota9x** [discord-tickets/bot#517] [Support (Q&A)] [answered]
Whenever I create a ticket using a panel, I get a "Something went wrong error" with identifier 65a48d5f40. Despite the error, the ticket opens and everything functions as normal, it's just annoying.

![Screenshot_20240114-204404](https://github.com/discord-tickets/bot/assets/105396031/ab0b76b9-e8d5-4f38-b015-79766d93f85c)

**Comment by eartharoid** [comment, marked as answer]
```
 15/01/24 01:41:52  [ERROR] (MODALS) "topic" modal execution error: Error: Spacetime: Cannot find timezone named: 'EST'. Please enter an IANA timezone id.
 ```

@brlxkn try `US/Eastern` or `America/New_York` for the working hours timezone.
